### PR TITLE
Add 36 port 100g sku for x86_64-arista_7800r3a_36d series of linecards.

### PR DESCRIPTION
**Pull request by skbarista**
Add 36 port 100g sku for x86_64-arista_7800r3a_36d series of linecards.

### CODE REVIEW

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# name         lanes             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet0      72,73,74,75       Ethernet1/1  1      Ext        100000      Eth0-ASIC0         1           1               8
+Ethernet8      80,81,82,83       Ethernet2/1  2      Ext        100000      Eth8-ASIC0         1           3               8
```
Comment (arlakshm) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini:3]: Why is the `corePortId`  not continous here ?

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/j2p-a7800r3a-36d-36x400G.config.bcm [hunk 2]
```diff
@@ -0,0 +1,984 @@
+soc_family=BCM8885X
+
```
Comment (arlakshm) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/j2p-a7800r3a-36d-36x400G.config.bcm:2]: `system_ref_core_clock_khz=1600000` is missing ?

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/port_config.ini [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# name         lanes             alias        index  role       speed      asic_port_name     coreId      corePortId      numVoq
```
Comment (arlakshm) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/port_config.ini:1]: Can we remove this file, it is not needed

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Ethernet224    56,57,58,59       Ethernet29/1 29     Ext        100000      Eth224-ASIC1       0           21              8
+Ethernet232    48,49,50,51       Ethernet30/1 30     Ext        100000      Eth232-ASIC1       0           23              8
+Ethernet240    40,41,42,43       Ethernet31/1 31     Ext        100000      Eth240-ASIC1       0           25              8
+Ethernet248    32,33,34,35       Ethernet32/1 32     Ext        100000      Eth248-ASIC1       0           27              8
+Ethernet256    24,25,26,27       Ethernet33/1 33     Ext        100000      Eth256-ASIC1       0           29              8
+Ethernet264    16,17,18,19       Ethernet34/1 34     Ext        100000      Eth264-ASIC1       0           31              8
+Ethernet272    8,9,10,11         Ethernet35/1 35     Ext        100000      Eth272-ASIC1       0           33              8
+Ethernet280    0,1,2,3           Ethernet36/1 36     Ext        100000      Eth280-ASIC1       0           35              8
+Ethernet-Rec1  221               Recirc0/0    19     Rec        400000      Rcy0-ASIC1         0           221             8
```
Comment (arlakshm) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini:20]: The index for `Ethernet-Rec1` and `Ethernet-IB1` is overlapping with the index for `Ethernet144` and `Ethernet-152`

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Ethernet80     56,57,58,59       Ethernet11/1 11     Ext        100000      Eth80-ASIC0        0           21              8
+Ethernet88     48,49,50,51       Ethernet12/1 12     Ext        100000      Eth88-ASIC0        0           23              8
+Ethernet96     40,41,42,43       Ethernet13/1 13     Ext        100000      Eth96-ASIC0        0           25              8
+Ethernet104    32,33,34,35       Ethernet14/1 14     Ext        100000      Eth104-ASIC0       0           27              8
+Ethernet112    24,25,26,27       Ethernet15/1 15     Ext        100000      Eth112-ASIC0       0           29              8
+Ethernet120    16,17,18,19       Ethernet16/1 16     Ext        100000      Eth120-ASIC0       0           31              8
+Ethernet128    8,9,10,11         Ethernet17/1 17     Ext        100000      Eth128-ASIC0       0           33              8
+Ethernet136    0,1,2,3           Ethernet18/1 18     Ext        100000      Eth136-ASIC0       0           35              8
+Ethernet-Rec0  221               Recirc0/0    19     Rec        400000      Rcy0-ASIC0         0           221             8
```
Comment (arlakshm) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini:20]: The index for Ethernet-Rec0 and Ethernet-IB0 is overlapping with the index for Ethernet144 and Ethernet-152

**Review by arlakshm — COMMENTED** [review]
as comments

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# name         lanes             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet144    72,73,74,75       Ethernet19/1 19     Ext        100000      Eth144-ASIC1       1           1               8
```
Comment (arlakshm) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini:2]: The 'asic_port_name' should start from Eth0 again.

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini (hunk 1 above)
Comment (skbarista) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini]: Fixed

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/port_config.ini (hunk 3 above)
Comment (skbarista) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/port_config.ini]: Fixed

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini (hunk 4 above)
Comment (skbarista) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini]: Fixed

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini (hunk 5 above)
Comment (skbarista) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini:20]: Fixed

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini (hunk 6 above)
Comment (skbarista) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini]: Fixed

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/j2p-a7800r3a-36d-36x400G.config.bcm (hunk 2 above)
Comment (skbarista) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/j2p-a7800r3a-36d-36x400G.config.bcm:2]: This is default for j2c+, we do not need to specify it explicitly in bcm conf file.

**Review by arlakshm — APPROVED** [review]

**Pull request merged**